test/spinach/cli_test.rb

```diff
@@ -257,6 +257,15 @@
       end
     end
 
+    describe 'when a particular feature list is passed with multiple lines' do
+      it 'returns the feature with the line numbers' do
+        cli = Spinach::Cli.new(['features/some_feature.feature:10:20'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.